src/game/server/swarm/asw_marine.cpp

```diff
@@ -4538,11 +4544,15 @@ void CASW_Marine::Suicide()
 	//SetNextThink(gpGlobals->curtime + 2.0f);
 }
 
+// called from BaseClass::Event_Killed
 bool CASW_Marine::BecomeRagdollOnClient( const Vector &force )
 {
 	if ( !CanBecomeRagdoll() ) 
 		return false;
 
+	if ( m_bSpawnZombineOnDeath )
```

Comment:
This should be part of `CanBecomeRagdoll`.

Reply:
like that?
![image](https://user-images.githubusercontent.com/95616101/219210892-9977764f-ee68-4f65-9869-d237781da871.png)

Reply:
yep